spec/std/isa/inst/Zvqdotq/vqdot.vv.yaml

```diff
@@ -0,0 +1,73 @@
+﻿# Copyright (c) Kallal Mukherjee.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../../schemas/inst_schema.json
+
+$schema: "inst_schema.json#"
+kind: instruction
+name: vqdot.vv
+long_name: Vector quad widening signed dot product (vector-vector)
```

Comment:
Spec has "Vector 8-bit Signed-Signed Dot Product"

For all of the instructions, take the base name from the spec directly (from "Synopsis", verbatim), then add the type clarification as you've done.